src/cpp/utils.cpp

```diff
@@ -0,0 +1,33 @@
+#include "utils.h"
+
+#include <fstream>
+#include <iomanip>
+
+namespace effil {
+
+Logger getLogger() {
+    static std::mutex lock;
+    static const char* logFile = getenv("EFFIL_LOG");
```

Comment:
Do we really need logging to file. It is very specific thing.

Reply:
I think that we need that. I have two points why:
1. We need logging. I think both of us agree with that
2. Logging into stdout not always possible:
    - when process is service
    - when process use stdout for its own needs